tests/casp/oci_keyboard.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Change keyboard layout
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    # Switch to UK
+    send_key 'alt-e';
+    send_key 'up';
+
+    # Check that UK layout is active
+    send_key 'alt-g';
```

Comment:
Yes, in general I think there should be a `wait_screen_change` around every send_key unless we ensure the expected result explicitly afterwards, e.g. `assert_screen`. But as you tested it already, I am ok with the as-is state :-)